### DISCUSSION TITLE
Crash after close Typecover

### DISCUSSION THREAD

**Discussion by ReiAccept** [badstorm/surface-pro-7-opencore#125] [Q&A]
Surface Pro7 i5 8G 128G version closes the Typecover for about 30 seconds and then open it. The screen cannot be turned on, but the keyboard light is on. After about five minutes, it will automatically restart. 

macOS version is "Ventrua 13.1"

EFI version is "Release 1887"

**Comment by ReiAccept** [comment]
After I update the OC to 0.8.8 and update some kexts the set "Hibernate Mode" to "NVRAM" in OpenCore ,It works good

However,after I fix it ,The system does not display the battery status

**Comment by pcborden98** [comment]
I'm having this issue as well on Ventura. It also occurs when removing the type cover (but oddly enough, not when putting it back on). Same exact Surface version.

**Comment by badstorm** [comment]
Since the conversation is dated I am closing the issue and moving it as a conversation.
